Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:14
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y chromium
+
+COPY . . 
+
+RUN npm ci --omit=dev \
+  && npm install prop-types \
```

Comment:
if prop-types is required for the build, can it be moved to dependencies?

Reply:
Looks like Next.js started requiring this lib at some point and nobody noticed. I just added to the Dockerfile so I wouldn't mess with the current deployment, but I guess it won't hurt to add it there